src/interactions/lennard_jones.jl

```diff
@@ -280,3 +290,163 @@ function potential(::LennardJonesSoftCore, r2, invr2, (σ2, ϵ, σ6_fac))
     six_term = σ2^3 * inv(r2^3 + σ2^3 * σ6_fac)
     return 4ϵ * (six_term ^ 2 - six_term)
 end
+
+
+@doc raw"""
+    AshbaughHatch(; cutoff, α, λ, p, use_neighbors, lorentz_mixing, weight_special,
+                         weight_solute_solvent, force_units, energy_units, skip_shortcut)
+
+The AshbaughHatch ($V_{\text{AH}}$) is a modified Lennard-Jones ($V_{\text{LJ}}$) 6-12 interaction between two atoms.
+
+The potential energy is defined by
+```math
+V_{\text{LJ}}(r_{ij}) = 4\varepsilon_{ij} \left[\left(\frac{\sigma_{ij}}{r_{ij}}\right)^{12} - \left(\frac{\sigma_{ij}}{r_{ij}}\right)^{6}\right] \\ 
+```
+
+```math
+V_{\text{AH}}(r_{ij}) =
```

Comment:
As it stands I think we don't apply the VLJ(rc) term by default? So either remove it in the docstring or add it to the code, in which case probably remove the `cutoff` option and don't use the cutoff path.